frontend/src/stores/useProjectStore.ts

```diff
@@ -99,7 +106,7 @@ interface ProjectStore {
   setName: (name: string) => void,
   createTransition: (transition: BaseAutomataTransition) => number,
   editTransition: (transition: Omit<BaseAutomataTransition, 'from' | 'to'>) => void,
-  createComment: (comment: ProjectComment) => number,
+  createComment: (comment: Omit<ProjectComment, 'id'>) => number,
```

Comment:
Is it worth doing something like this for `createTransition` as well? As opposed to casting the parameter when it is called (in `frontend/src/hooks/useTransitionCreation.ts`), then it will be consistent with the other functions and I also find the Omit typing to be nice and clear as to what is missing (so nice job with that 😁)

Reply:
Think I did it locally but forgot to commit 🤔 either way it's good idea so will do 🫡